<ui version="4.0" >
 <class>AddressWindow</class>
 <widget class="QMainWindow" name="AddressWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Addresses</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="resources/MIM.qrc" >:/icons/MIM.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QTabWidget" name="tabWidget" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>251</height>
     </rect>
    </property>
    <property name="tabPosition" >
     <enum>QTabWidget::South</enum>
    </property>
    <property name="currentIndex" >
     <number>0</number>
    </property>
    <widget class="QWidget" name="basicTab" >
     <attribute name="title" >
      <string>Address</string>
     </attribute>
     <widget class="QLineEdit" name="provinceBox" >
      <property name="geometry" >
       <rect>
        <x>112</x>
        <y>134</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="postalCodeLabel" >
      <property name="geometry" >
       <rect>
        <x>1</x>
        <y>159</y>
        <width>105</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Postal/Zip Code</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cityBox" >
      <property name="geometry" >
       <rect>
        <x>112</x>
        <y>108</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="cityLabel" >
      <property name="geometry" >
       <rect>
        <x>1</x>
        <y>107</y>
        <width>105</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>City</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="postalCodeBox" >
      <property name="geometry" >
       <rect>
        <x>112</x>
        <y>160</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="provinceLabel" >
      <property name="geometry" >
       <rect>
        <x>1</x>
        <y>133</y>
        <width>105</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Province/State</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame" >
      <property name="geometry" >
       <rect>
        <x>112</x>
        <y>53</y>
        <width>131</width>
        <height>48</height>
       </rect>
      </property>
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTextEdit" name="streetAddressBox" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>130</width>
         <height>45</height>
        </rect>
       </property>
       <property name="tabChangesFocus" >
        <bool>true</bool>
       </property>
       <property name="acceptRichText" >
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="countryBox" >
      <property name="geometry" >
       <rect>
        <x>112</x>
        <y>186</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="countryLabel" >
      <property name="geometry" >
       <rect>
        <x>1</x>
        <y>185</y>
        <width>105</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Country</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="firstNameBox" >
      <property name="geometry" >
       <rect>
        <x>141</x>
        <y>2</y>
        <width>100</width>
        <height>18</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget" >
      <property name="geometry" >
       <rect>
        <x>249</x>
        <y>2</y>
        <width>89</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="spouseNameLabel" >
         <property name="text" >
          <string>Spouse Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="homePhoneLabel" >
         <property name="text" >
          <string>Home Phone</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="workPhoneLabel" >
         <property name="text" >
          <string>Work Phone</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="workExtensionLabel" >
         <property name="text" >
          <string>Extension</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="faxLabel" >
         <property name="text" >
          <string>Fax Number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="cellPhoneLabel" >
         <property name="text" >
          <string>Cell Phone</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="titleBox" >
      <property name="geometry" >
       <rect>
        <x>112</x>
        <y>2</y>
        <width>28</width>
        <height>18</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget" >
      <property name="geometry" >
       <rect>
        <x>342</x>
        <y>2</y>
        <width>131</width>
        <height>152</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLineEdit" name="spouseNameBox" />
       </item>
       <item>
        <widget class="QLineEdit" name="homePhoneBox" >
         <property name="inputMask" >
          <string># (999) 999-9999; </string>
         </property>
         <property name="text" >
          <string> () -</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="workPhoneBox" >
         <property name="inputMask" >
          <string># (999) 999-9999; </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="workExtensionBox" />
       </item>
       <item>
        <widget class="QLineEdit" name="faxBox" >
         <property name="inputMask" >
          <string># (999) 999-9999; </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="cellPhoneBox" >
         <property name="inputMask" >
          <string># (999) 999-9999; </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="lastNameBox" >
      <property name="geometry" >
       <rect>
        <x>112</x>
        <y>28</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lastNameLabel" >
      <property name="geometry" >
       <rect>
        <x>1</x>
        <y>27</y>
        <width>105</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Last Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="streetAddressLabel" >
      <property name="geometry" >
       <rect>
        <x>1</x>
        <y>53</y>
        <width>105</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text" >
       <string>Street Address</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="firstNameLabel" >
      <property name="geometry" >
       <rect>
        <x>1</x>
        <y>1</y>
        <width>105</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>First Name</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="isBusinessBox" >
      <property name="geometry" >
       <rect>
        <x>98</x>
        <y>2</y>
        <width>16</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="extendedTab" >
     <attribute name="title" >
      <string>Extended</string>
     </attribute>
     <widget class="QWidget" name="gridLayout_2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>221</width>
        <height>120</height>
       </rect>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="3" column="0" >
        <widget class="QLabel" name="anniversaryLabel" >
         <property name="text" >
          <string>Anniversary</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="urlLabel" >
         <property name="text" >
          <string>Website</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="emailLabel" >
         <property name="text" >
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="urlBox" />
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="birthdateLabel" >
         <property name="text" >
          <string>Birthdate</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="emailBox" />
       </item>
       <item row="2" column="1" >
        <widget class="QDateEdit" name="birthdateBox" >
         <property name="specialValueText" >
          <string/>
         </property>
         <property name="displayFormat" >
          <string>yyyy/M/d</string>
         </property>
         <property name="calendarPopup" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QDateEdit" name="anniversaryBox" >
         <property name="specialValueText" >
          <string/>
         </property>
         <property name="displayFormat" >
          <string>yyyy/M/d</string>
         </property>
         <property name="calendarPopup" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTextEdit" name="notesBox" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>139</y>
        <width>241</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="notesLabel" >
      <property name="geometry" >
       <rect>
        <x>3</x>
        <y>122</y>
        <width>46</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text" >
       <string>Notes</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayout_3" >
      <property name="geometry" >
       <rect>
        <x>230</x>
        <y>0</y>
        <width>241</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>3</number>
       </property>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="supportBox" >
         <property name="toolTip" >
          <string>Pledged amount per period in donor's currency.</string>
         </property>
         <property name="statusTip" >
          <string>Pledged amount per period in donor's currency.</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="supportLabel" >
         <property name="toolTip" >
          <string>Pledged average monthly amount in donor's currency.</string>
         </property>
         <property name="statusTip" >
          <string>Pledged average monthly amount in donor's currency.</string>
         </property>
         <property name="text" >
          <string>Support</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QLabel" name="monthlyLabel" >
         <property name="toolTip" >
          <string>Montly amount in your currency.</string>
         </property>
         <property name="statusTip" >
          <string>Montly amount in your currency.</string>
         </property>
         <property name="text" >
          <string>Monthly</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="currencyLabel" >
         <property name="toolTip" >
          <string>Donor's currency</string>
         </property>
         <property name="statusTip" >
          <string>Donor's currency</string>
         </property>
         <property name="text" >
          <string>Currency</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" >
        <widget class="QLineEdit" name="monthlyCalcBox" >
         <property name="toolTip" >
          <string>Montly amount in your currency.</string>
         </property>
         <property name="statusTip" >
          <string>Montly amount in your currency.</string>
         </property>
         <property name="maxLength" >
          <number>32767</number>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight</set>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QLabel" name="periodLabel" >
         <property name="toolTip" >
          <string>Annual amount in your currency.</string>
         </property>
         <property name="statusTip" >
          <string>Annual amount in your currency.</string>
         </property>
         <property name="text" >
          <string>Times/yr</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3" >
        <widget class="QLineEdit" name="periodBox" >
         <property name="toolTip" >
          <string>Pledged donations per year (period).</string>
         </property>
         <property name="statusTip" >
          <string>Pledged donations per year (period).</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QComboBox" name="currencyBox" >
         <property name="toolTip" >
          <string>Donor's currency</string>
         </property>
         <property name="statusTip" >
          <string>Donor's currency</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QListView" name="categoryList" >
      <property name="geometry" >
       <rect>
        <x>260</x>
        <y>110</y>
        <width>211</width>
        <height>70</height>
       </rect>
      </property>
      <property name="toolTip" >
       <string>Categories</string>
      </property>
      <property name="statusTip" >
       <string>Categories</string>
      </property>
      <property name="horizontalScrollBarPolicy" >
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="editTriggers" >
       <set>QAbstractItemView::DoubleClicked</set>
      </property>
      <property name="uniformItemSizes" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="newCategoryCombo" >
      <property name="geometry" >
       <rect>
        <x>260</x>
        <y>182</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip" >
       <string>Add a category</string>
      </property>
      <property name="statusTip" >
       <string>Add a category</string>
      </property>
      <property name="editable" >
       <bool>true</bool>
      </property>
      <property name="insertPolicy" >
       <enum>QComboBox::NoInsert</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="addCategoryButton" >
      <property name="geometry" >
       <rect>
        <x>422</x>
        <y>183</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip" >
       <string>Add Category</string>
      </property>
      <property name="statusTip" >
       <string>Add Category</string>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="icon" >
       <iconset resource="resources/MIM.qrc" >:/icons/add.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteCategoryButton" >
      <property name="geometry" >
       <rect>
        <x>451</x>
        <y>183</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip" >
       <string>Delete Category</string>
      </property>
      <property name="statusTip" >
       <string>Delete Category</string>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="icon" >
       <iconset resource="resources/MIM.qrc" >:/icons/delete.png</iconset>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>Toolbar</string>
   </property>
   <property name="accessibleName" >
    <string>Toolbar</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
  </widget>
 </widget>
 <tabstops>
  <tabstop>firstNameBox</tabstop>
  <tabstop>lastNameBox</tabstop>
  <tabstop>streetAddressBox</tabstop>
  <tabstop>cityBox</tabstop>
  <tabstop>provinceBox</tabstop>
  <tabstop>postalCodeBox</tabstop>
  <tabstop>countryBox</tabstop>
  <tabstop>spouseNameBox</tabstop>
  <tabstop>homePhoneBox</tabstop>
  <tabstop>workPhoneBox</tabstop>
  <tabstop>workExtensionBox</tabstop>
  <tabstop>faxBox</tabstop>
  <tabstop>cellPhoneBox</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="resources/MIM.qrc" />
 </resources>
 <connections/>
</ui>
